include/xrpl/protocol/jss.h

```diff
@@ -55,10 +55,13 @@ JSS(AssetClass);           // in: Oracle
 JSS(AssetPrice);           // in: Oracle
 JSS(AuthAccount);          // in: AMM Auction Slot
 JSS(AuthAccounts);         // in: AMM Auction Slot
+JSS(Authorize);            // in: account_permission
 JSS(BaseAsset);            // in: Oracle
 JSS(BidMax);               // in: AMM Bid
 JSS(BidMin);               // in: AMM Bid
 JSS(ClearFlag);            // field.
+JSS(DelegateTicketSequence); // in: Delegate transactions
+JSS(DelegateSequence);     // in: Delegate transactions
```

Comment:
we dont need these here if already defined in `sfield.macro`. the JSON field can be called using `sfDelegateTicketSequence.jsonName` and `sfDelegateSequence.jsonName`